docs: add code along with the first recipe to `working-with-files` (#65)
rostools/github-intro: sessions/working-with-files.qmd

{{< text_snippet sticky_up >}}

> ## Add a soup recipe to your repository

Author: ```suggestion
## Add a soup recipe
```

{{< text_snippet sticky_up >}}

> ## Add a soup recipe to your repository

Reviewer: In general, code along sections are "we do" and exercises are "you do". Could you revise the language here to match the "we do" for this section?

Author: Ah! I was thinking about how we use "you" and "we". Thanks for clarifying. Will do!

> Now, it's time to create your first file on GitHub. Let's add a soup
> recipe :bowl_with_spoon:.

Author: ```suggestion
Now, we'll create our first file on GitHub. Let's add a soup
recipe :bowl_with_spoon:.
```

> Now, it's time to create your first file on GitHub. Let's add a soup
> recipe :bowl_with_spoon:.
> 
> In the "Code" tab of your repository, click the button saying "Add file"
> in the top-right side of the page. In the dropdown menu, click "Create
> new file".
> 
> This leads us to a new page with an empty file without any content.
> 
> In the "Name your file..." field, in the top-left corner, type the name
> of your file. Let's follow the good naming practices we've read about
> above and call it `soup.md`.
> 
> For the content of the file, copy and paste the following recipe:
> 

Author: ```suggestion
Now, we'll time to create our first file on GitHub. Let's add a soup
recipe :bowl_with_spoon:.

In the "Code" tab of the repository, we'll click the button saying "Add file"
in the top-right side of the page. In the dropdown menu, we'll click "Create
new file".

This leads us to a new page with an empty file without any content.

In the "Name your file..." field, in the top-left corner, we'll type the name
of our file. Let's follow the good naming practices we've read about
above and call it `soup.md`.

For the content of the file, we'll copy and paste the following recipe:

```

``` markdown
Ingredients for tomato soup 

- 4 tbs butter
- 0.5 large onion, cut into large wedges
- 1 can of tomatoes
- 1,5 cups water, vegetable or chicken stock
- 0.5 tsp salt, or more to taste

Instructions

1. Melt the butter over medium heat in a large saucepan.
2. Add onion wedges, water, tomatoes with their juices, and salt.
3. Bring to a simmer and cook for about 40 minutes. Stir occasionally.
4. Blend the soup, and season to taste.
```

> Once you have added the file name and the recipe itself, click the green
> "Commit changes" button in the top right corner of the page.
> 
> Write a descriptive commit message like "Add tomato soup recipe". You
> don't need to write an extended description. Keep the "Commit directly
> to the `main` branch" option selected.

Author: ```suggestion
Once we have added the file name and the recipe itself, let's save the changes by clicking the green
"Commit changes" button in the top right corner of the page.

As the commit message, we'll write "Add tomato soup recipe". We
don't need to write an extended description. We'll keep the "Commit directly
to the `main` branch" option selected, then click the "Commit changes" button.
```

> to the `main` branch" option selected.
> 
> Click "Commit changes".

Reviewer: ```suggestion
to the `main` branch" option selected and then we'll click the "Commit changes" button.
```

Author: Added in suggestion above.

> You have now added the first recipe to your repository! :tada:
> 
> Now, you'll practice creating files in your repository by adding another
> recipe.
> 

Reviewer: ```suggestion
We've now added our first recipe to the repository! :tada: Now, you'll practice creating files in your repository by adding another
recipe.

```

## :woman_technologist: Exercise: Add a cookie recipe to your repository

**Time: \~10 minutes**

> Let's add the second recipe to your repository: A cookie recipe!

Author: ```suggestion
Now you'll add the second recipe to your repository: A cookie recipe!
```
